basilisk/ipc/windows_ipc.py

```
@@ -0,0 +1,145 @@
+				)
+
+				if pipe_handle == win32file.INVALID_HANDLE_VALUE:
+					logger.error("Failed to create named pipe")
+					break
+
+				# Wait for a client to connect
+				logger.debug("Waiting for client connection...")
+				win32pipe.ConnectNamedPipe(pipe_handle, None)
+
+				# Handle the connection in a separate thread
+				def handle_connection(pipe_handle):
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)